proto/internal/temporal/server/api/enums/v1/task.proto

```diff
@@ -75,6 +75,12 @@ enum TaskType {
 
     TASK_TYPE_REPLICATION_SYNC_HSM = 30;
     TASK_TYPE_REPLICATION_SYNC_VERSIONED_TRANSITION = 31;
+
+    // A task that applies a batch of state changes to a CHASM entity.
+    TASK_TYPE_CHASM_PURE = 32;
+
+    // An immediate task generated by a CHASM component.
+    TASK_TYPE_CHASM_TRANSFER = 33;
```

Comment:
Not necessarily a transfer task right? The same physical task could be in timer/outbound queue as well.

I mean we need to support the case where chasm developers adding delayed tasks with side effect.